ggml/src/ggml-vulkan/vulkan-shaders/dequant_iq3_s.comp

```diff
@@ -22,14 +22,15 @@ void main() {
     const uint b_idx = 256 * ib + 32 * is;
 
     const float d = float(data_a[ib].d);
-    const float db = d * (1 + 2 * ((data_a[ib].scales[is] >> (4 * (is % 2))) & 0xf));
+    const float db = d * (1 + 2 * ((data_a[ib].scales[is / 2] >> (4 * (is % 2))) & 0xf));
 
     // We must produce 32 values using 4 sign bytes, 1 qh byte, 8 qs bytes.
     uint qh = data_a[ib].qh[is];
     [[unroll]] for (uint l = 0; l < 8; ++l) {
-        uint qs = data_a[ib].qs[8 * is + l];
+        uint iqs = 8 * is + l;
```

Comment:
Not sure why this wasn't done here, but let's mark these variables as const, similar to the other functions you fixed.